### PR TITLE
Functions

### PR DESCRIPTION
**Pull request by kooshan75**

### CODE REVIEW

File: 1-js/02-first-steps/14-function-basics/3-min/task.md [hunk 1]
```diff
@@ -2,11 +2,12 @@ importance: 1
 
 ---
 
-# Function min(a, b)
+# تابع min(a, b)
 
 Write a function `min(a,b)` which returns the least of two numbers `a` and `b`.
```
Comment (mehradsadeghi) [inline 1-js/02-first-steps/14-function-basics/3-min/task.md]: This line should be removed. Correct me if I'm wrong.

File: 1-js/02-first-steps/14-function-basics/4-pow/task.md [hunk 2]
```diff
@@ -13,7 +13,9 @@ pow(1, 100) = 1 * 1 * ...* 1 = 1
 ```
 
 Create a web-page that prompts for `x` and `n`, and then shows the result of `pow(x,n)`.
```
Comment (mehradsadeghi) [inline 1-js/02-first-steps/14-function-basics/4-pow/task.md]: This line should be removed. Correct me if I'm wrong.

File: 1-js/02-first-steps/14-function-basics/4-pow/task.md [hunk 3]
```diff
@@ -13,7 +13,9 @@ pow(1, 100) = 1 * 1 * ...* 1 = 1
...
+یک صفحه وب که دو مقدار `x` و `n` را می‌گیرد و جواب `pow(x,n)` را بر‌می‌گرداند را بسازید.
 
 [demo]
 
 P.S. In this task the function should support only natural values of `n`: integers up from `1`.
```
Comment (mehradsadeghi) [inline 1-js/02-first-steps/14-function-basics/4-pow/task.md]: This line should be removed. Correct me if I'm wrong.

File: 1-js/02-first-steps/14-function-basics/article.md [hunk 4]
```diff
@@ -322,80 +316,80 @@ alert( doNothing() === undefined ); // true
...
+## نامگذاری یک تابع [#function-naming]
 
-It is a widespread practice to start a function with a verbal prefix which vaguely describes the action. There must be an agreement within the team on the meaning of the prefixes.
+توابع، اعمال هستند. بنابراین اسم آنها عموما یک فعل است. باید خلاصه باشد و با بیشترین دقت ممکن، فعالیت تابع را توصیف کند که اگر کسی کد را مطالعه می‌کرد متوجه نوع فعالیت تابع بشود.
 
-For instance, functions that start with `"show"` usually show something.
+این یک روش معمول است که تابع را با پیشوند فعلی شروع کنیم که کارش را به گنگی توصیف کند. یک توافقی در تیم باید بر معنی‌های این پیشوندها باشد.
 
+برای نمونه، توابعی که با `"show"` شروع میشوند، معمولی چیزی را نمایش می‌دهند.
 Function starting with...
```
Comment (mehradsadeghi) [inline 1-js/02-first-steps/14-function-basics/article.md]: This line should be removed. Correct me if I'm wrong.

Comment (kooshan75) [reply 1-js/02-first-steps/14-function-basics/article.md]: Oops! They must've been slipped through my hand! Sorry. Fixed them.

**Pull request merged**